### ISSUE TITLE
Versioning guidance

### ISSUE THREAD

**Issue by raissameyer** [BeBOP-OBON/0_protocol_collection_template#10]
From the BeBOP meeting 2022-12-07

If you tweak somebody else's protocol (however small) under a different team of authors, it should be a new protocol, not a new version of the same protocol. If our system is used well, these scenarios could also be solved by forking somebody else's protocol.

**Comment by lukenoaa** [comment]
I agree it should be a fork. Is it sufficient to list the parent protocol under the "RELATED PROTOCOLS" section? Additionally, the fork information will provide provenance. I think we can close this issue.
